### ISSUE TITLE
LineChart tickValues not working

### ISSUE THREAD

**Issue by luckyrass** [microsoft/fluentui#15787]
### Environment Information

- **Package version(s)**: (5.0.0-beta.4)
- **Browser and OS versions**: (All)

### Please provide a reproduction of the bug in a codepen:

Code to repro it:
Set tickValues={[new Date('2020-01-03'), new Date('2020-03-09')]}, It is expected that the start date in X-axis is 2020-01-03.

```
import * as React from 'react';
import { IChartProps, ILineChartProps, LineChart } from '@fluentui/react-charting';
import { DefaultPalette } from '@fluentui/react/lib/Styling';

interface ILineChartBasicState {
  width: number;
  height: number;
}

export class LineChartBasicExample extends React.Component<{}, ILineChartBasicState> {
  constructor(props: ILineChartProps) {
    super(props);
    this.state = {
      width: 700,
      height: 300,
    };
  }

  public render(): JSX.Element {
    return <div>{this._basicExample()}</div>;
  }

  private _onWidthChange = (e: React.ChangeEvent<HTMLInputElement>) => {
    this.setState({ width: parseInt(e.target.value, 10) });
  };
  private _onHeightChange = (e: React.ChangeEvent<HTMLInputElement>) => {
    this.setState({ height: parseInt(e.target.value, 10) });
  };

  private _basicExample(): JSX.Element {
    const data: IChartProps = {
      chartTitle: 'Line Chart',
      lineChartData: [
        {
          legend: 'From_Legacy_to_O365',
          data: [
            {
              x: new Date('2020-03-08T00:00:00.000Z'),
              y: 300000,
              onDataPointClick: () => alert('click on 300000'),
            },
            {
              x: new Date('2020-03-09T00:00:00.000Z'),
              y: 218000,
              onDataPointClick: () => alert('click on 218000'),
            },
          ],
          color: DefaultPalette.blue,
          onLineClick: () => console.log('From_Legacy_to_O365'),
        },
        {
          legend: 'All',
          data: [
            {
              x: new Date('2020-01-03T00:00:00.000Z'),
              y: 0,
            },
            {
              x: new Date('2020-03-08T00:00:00.000Z'),
              y: 300000,
            },
            {
              x: new Date('2020-03-09T00:00:00.000Z'),
              y: 298000,
            },
          ],
          color: DefaultPalette.green,
        },
      ],
    };

    const rootStyle = { width: `${this.state.width}px`, height: `${this.state.height}px` };
    const margins = { left: 35, top: 20, bottom: 35, right: 20 };

    return (
      <>
        <label>change Width:</label>
        <input type="range" value={this.state.width} min={200} max={1000} onChange={this._onWidthChange} />
        <label>change Height:</label>
        <input type="range" value={this.state.height} min={200} max={1000} onChange={this._onHeightChange} />
        <div style={rootStyle}>
          <LineChart
            data={data}
            legendsOverflowText={'Overflow Items'}
            yMinValue={200}
            yMaxValue={301}
            height={this.state.height}
            width={this.state.width}
            margins={margins}
            tickFormat={'%m/%d'}
            tickValues={[new Date('2020-01-03'), new Date('2020-03-09')]}
          />
        </div>
      </>
    );
  }
}

```

#### Actual behavior:
Start date in X-axis is 03/08
![image](https://user-images.githubusercontent.com/12230173/97666031-c3212a00-1ab7-11eb-833f-d4436265bb3e.png)

Start date in X-axis is 01/02
#### Expected behavior:
![image](https://user-images.githubusercontent.com/12230173/97666346-64a87b80-1ab8-11eb-9fd2-e2a9ce3aa6f7.png)

### Priorities and help requested:

Are you willing to submit a PR to fix? (Yes, No)

Requested priority: (Blocking, High, Normal, Low)
Normal

Products/sites affected: (if applicable)
https://admin-ignite.microsoft.com/Adminportal/Home?source=applauncher#/reportsUsage

**Comment by paulgildea** [comment]
@luckyrass Thanks for filing this issue!

@Raghurk FYI

**Comment by jameelakowsar** [comment]
@luckyrass ,

Issue fixed and version released @fluentui/react-charting@v5.0.0-beta.18.
Can you please confirm and close work item?
